docs/api/api/api.txt

```diff
@@ -151,6 +151,8 @@ XmlResult: status
 
 Parameters:
   cmd: change_password
+  cmd: lock             # will lock the user and his home projects
+  cmd: delete           # will mark the user as deleted and remove her home projects
```

Comment:
JFYI, a user delete is not really possible, because the user name gets stored also in the various histories which we must not touch (or we wouldn't be able to review past changes propperly anymore). So we want to avoid that someone else can take over the same name.